.github/ISSUE_TEMPLATE/bug-issue.yml

```diff
@@ -1,6 +1,6 @@
 name: 🐞 Bug report
 description: Report a very clearly broken issue.
-title: 'bug: <title>'
+title: 'bug(scope): <title>'
 labels: [bug]
```

Comment:
Most people opening issues get confused by the title syntax already as is. Do you think it's a good idea to add this?

Reply:
In that case I think using `(patch)` fits better, as mostly, bugs exist in patches. In fact, the same theory is applied to the Patch Request template where its default is `(patch)` as well.

Reply:
usually it's feat(<patch app>): add <patch name>. For bugs I still think using "bug:" soley would be better than what is done currently as some people put their title in the <> placeholder brackets

Reply:
I get your point, but the same would have happened to the Patch Request issue template then, yeah? Perhaps trying `bug(patch): <title>` is an idea. If you notice people indeed don't comply, you can undo it again.

Reply:
> but the same would have happened to the Patch Request issue template then, yeah

Yeah, it would be "feat:".

> Perhaps trying `bug(patch): <title>`

This will cause the following titles:

-  bug(hide ads in app): <hide ads in app>
- bug(patch): <hide ads>
- bug(hide ads in app): <title>
- bug(patch): hide ads

Instead, if we go with `bug:` the most likely title will be `bug: hide ads for app` which is correct.

Reply:
> Yeah, it would be "feat:".

Sooo, if I understand correctly, my PR should also remove the scope (`(patch)`) for consistency. If you want it to be generic, I'll push a change that makes it effectively

- `bug: `
- `feat: `

Agreed?

Reply:
That sounds good